src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -38,4 +38,16 @@ export default class PaypalCommerceRequestSender {
 
         return res.body as OrderData;
     }
+
+    async getOrderStatus(): Promise<OrderStatus> {
+        const url = `/api/storefront/initialization/paypalcommerce`;
```

Comment:
there's no variable in this string, you should be able to use simple quotes `'`

Reply:
fixed